src/main/flight/pid.c

```diff
@@ -132,12 +132,12 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .dterm_notch_hz = 0,
         .dterm_notch_cutoff = 0,
         .dterm_filter_type = FILTER_PT1,
-        .itermWindupPointPercent = 100,
+        .itermWindupPointPercent = 40,
```

Comment:
And why do we want this back now? iterm relax will be enabled on all targets in 4.0. But I think you do it in case of crash?

Reply:
Yes just for safety.  Lets get good pilots to test at 100 in the RC phase, then if the feedback is good, lock it in at 100 for 4.0.